cmake build fails on commit b562211

When building on Ubuntu 18 with CMake, I get the following error:

(base) ryan@ryan-desktop:~/DR3/drake$ cmake -DCMAKE_BUILD_TYPE="Release" && make -j4
-- The C compiler identification is GNU 7.5.0
-- The CXX compiler identification is GNU 7.5.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Bazel: /usr/bin/bazel (found suitable version "4.0.0", minimum required is "3.0") 
CMake Error at CMakeLists.txt:229 (message):
  Could NOT find the python3 executable

-- Configuring incomplete, errors occurred!
See also "/home/ryan/DR3/drake/CMakeFiles/CMakeOutput.log".

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -200,10 +200,6 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_STANDARD 17)
 
-# The paths within which to search for the python3 executable should match the
-# interpreter paths for the Python runtimes in the Python toolchains
-# registered by Bazel in WORKSPACE.
-#
 # The supported Python major/minor versions should match those listed in both
 # doc/developers.rst and tools/workspace/python/repository.bzl.
 if(APPLE)
@@ -221,10 +217,7 @@ else()
 endif()
 
 if(CMAKE_VERSION VERSION_LESS 3.12)
-  find_program(PYTHON_EXECUTABLE NAMES python3
-    PATHS "${FIND_PROGRAM_PYTHON3_PATHS}"
-    NO_DEFAULT_PATH
-  )
+  find_program(PYTHON_EXECUTABLE NAMES "${FIND_PROGRAM_PYTHON3_NAMES}")
   if(NOT PYTHON_EXECUTABLE)
     message(FATAL_ERROR "Could NOT find the python3 executable")
   endif()

